Code seems to fail to compile:

```
../../src/app/server/Dnssd.cpp:157:5: error: expected unqualified-id before 'if'
  157 |     if (mICDManager->SupportsFeature(Clusters::IcdManagement::Feature::kLongIdleTimeSupport))
      |     ^~
../../src/app/server/Dnssd.cpp:169:5: error: 'advParams' does not name a type
  169 |     advParams.SetICDModeToAdvertise(ICDModeToAdvertise);
  ```
  
  @wqx6

--- a/src/app/server/Dnssd.cpp
+++ b/src/app/server/Dnssd.cpp
@@ -149,8 +149,9 @@ CHIP_ERROR DnssdServer::SetEphemeralDiscriminator(Optional<uint16_t> discriminat
 template <class AdvertisingParams>
 void DnssdServer::AddICDKeyToAdvertisement(AdvertisingParams & advParams)
 {
-VerifyOrReturn(mICDManager != nullptr, ChipLogError(Discovery, "Invalid pointer to the ICDManager which is required for adding Dnssd advertisement key"));
-    }
+    VerifyOrReturn(
+        mICDManager != nullptr,
+        ChipLogError(Discovery, "Invalid pointer to the ICDManager which is required for adding Dnssd advertisement key"));
 
     Dnssd::ICDModeAdvertise ICDModeToAdvertise = Dnssd::ICDModeAdvertise::kNone;
     // Only advertise the ICD key if the device can operate as a LIT

